backend/src/CardHandler.ts

```diff
@@ -0,0 +1,373 @@
+import { GameCard, CardType } from './types/Cards';
+import { SourceKind, PickCardEvent, PlayerUpdateEvent } from './types/Events';
```

Comment:
nit: alphabetize these imports (PickCardEvent, PlayerUpdateEvent, SourceKind)

also alphabetize the files across imports